platform/o.n.core/src/org/netbeans/core/ui/ProductInformationPanel.form

```diff
@@ -90,13 +98,15 @@
         </Component>
       </SubComponents>
     </Container>
-    <Container class="javax.swing.JScrollPane" name="jScrollPane3">
+    <Container class="javax.swing.JScrollPane" name="copyRightScrollPane">
```

Comment:
(very) minor niggle - should be `copyrightScrollPane`

Reply:
oh, yeah i can fix that :)